src/terminologySectionPanel/codesSection/useCodeSystems.ts

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
+import useTerminologyServiceApi, {
+  CodeSystem,
+} from "../../api/useTerminologyServiceApi";
+
+export function useCodeSystems() {
+  const [codeSystems, setCodeSystems] = useState<CodeSystem[]>([]);
+  // const [loading, setLoading] = useState(true); maybe add later since this call is slow
+
+  useEffect(() => {
+    const fetchCodeSystems = async () => {
+      // eslint-disable-next-line
```

Comment:
we remove these comments right ?

Reply:
No, it breaks the build

Reply:
the api class isn't actually an api class. It's a promise. 
The issue being thrown is a linter issue based off of naming conventions and the way we structure the api class. 

I tried a few variations of renaming files, functions, etc. But it appears to be completely just based on naming conventions. 
There didn't seem to be a way to fix this without completely rewriting the terminology api class to no longer be a promise.